components/atoms/TocWrapper/TocWrapperStyles.ts

```diff
@@ -1,14 +1,24 @@
 import styled from 'styled-components'
 
 interface Props {
-    anyHeaders?: boolean
+    isHighlighted?: boolean
 }
 
+export const StyledTocTopWrapper = styled.div`
+    position: relative;
+`
+
 export const StyledTocWrapper = styled.div`
+    position: absolute;
+    left: -5rem;
```

Comment:
@ltagliaferri I've moved the ToC to the left to accommodate the longer ToC contents (for example on `regular-expression-patterns`) — is that something you would prefer to leave as is (with the ToC covering parts of the header/post)? We can always ensure its position with a z index.

Reply:
The ToC is getting cut off in my views

Reply:
The breakpoint in `b28eefd` should resolve that. `1228px` looks like this
<img width="1218" alt="Screen Shot 2021-08-25 at 11 46 09 AM" src="https://user-images.githubusercontent.com/43454804/130822521-3f6a5237-874f-4d93-8aca-edd41ce4c387.png">